### DISCUSSION TITLE
把電腦穿在身上，美軍與 MIT 合作研發智慧型纖維

### DISCUSSION THREAD

**Discussion by 239216** [1101-GS4719-C/Python-Community-GS4719-C#108] [General]
美國陸軍戰鬥能力發展司令部與 MIT、產業界共同創立的「士兵奈米科技研究所將最新的研究發表在科學期刊《自然─通訊》。

新的人工聚合物纖維，內含數百個微型晶片，傳輸距離可達數十公尺，並可用針線縫合在任何布料上，經過十多次水洗後仍可正常運作。目前的試驗成品，已可儲存約 767 千位元的短片，以及 0.48 MB 的音樂檔案，無電源狀態下，資料可保存兩個月以上。
也可與 AI 連結，提供士兵的相關數據，包括心律、呼吸頻率、肌肉狀態等，使 AI 快速辨識士兵所處的環境以及身體狀態。陸軍測試成功讓智慧纖維與 AI 收集並計算約 270 分鐘長的生理數據，並藉此訓練 AI 演算法，最後 AI 與智慧纖維搭配，可快速且精準顯示士兵正在從事何種活動，以及周遭環境的分析，準確率高達 96%。
https://technews.tw/2021/06/18/us-army-and-mit-developing-smart-fiber-that-turns-clothes-into-wearable-computers/

**Comment by 109802503gloria** [comment]
記得智慧衣的發展已經有一段時間了，像台灣在2016年曾經研發過iSmartweaR，但是使用雷達感測技術，也只能傳輸即時資料。

**Comment by chupochun** [comment]
若是此項科技運用在生活上，對普通人肯定很有幫助

**Comment by xkllkx** [comment]
原來還有這種應用阿，謝謝樓主分享，希望能夠盡早把成本壓低，盡快普及

**Comment by awkward-willy** [comment]
這樣的技術或許能在未來帶來巨大的改變
不如穿戴式裝置或行動裝置的限制
也不向植入晶片那樣的風險性
直接穿在身上是相當方便的選擇

**Comment by treeann** [comment]
沒想到能有這樣的運用 謝謝樓主分享

**Comment by ghost** [comment]
原來有這回事== 感覺天花板挺高的

**Comment by Lai27330** [comment]
感覺也很適合應用在老年人照護的方面，可以比穿戴式裝置更及時的提供協助。

**Comment by etin2468** [comment]
文中提到水洗十多次沒問題，但軍人的衣物應該會更常洗吧，這樣真的沒問題?

**Comment by bloodtrail0511** [comment]
不清楚除了水洗之外，如果摔倒或刮傷布料會不會對資料的存儲有影響

**Comment by stan890425** [comment]
像這種材料就得仰賴新的材料纖維，
而不能使用舊的人造纖維了~

**Comment by yi-1111** [comment]
但是這種技術，最怕被敵人竊取

**Comment by liangyinglly** [comment]
通常會使用到這種可與 AI 連結的新智慧纖維，來量測心律、呼吸頻率、肌肉狀態的人，會是健身運動的人或士兵，會大量出汗，因此我認為耐洗承受度相當重要，值得未來發展進步。

**Comment by sys511613** [comment]
這個技術如果應用在運動員好像也不錯

**Comment by leon10456** [comment]
原來有這回事 感謝分享

**Comment by Jimmy911218** [comment]
在電影產業也有看到類似的科技超酷的ˊ
